src/scripts/main.js

```diff
@@ -415,8 +415,47 @@ function setIntegrationStackSupport(value) {
 	integrationStackSupport = value;
 }
 
+function drop(event) {
+	event.preventDefault();
+	var data;
+	var msie = document.documentMode;
+	if (msie) {
+		data = event.dataTransfer.getData("Text");
+	} else {
+		data = event.dataTransfer.getData("text/uri-list");
+	}
+	var DOWNLOAD_JBOSS_ORG_JBOSSTOOLS_CENTRAL_INSTALL_CONNECTORS = "http://download.jboss.org/jbosstools/central/install?connectors=";
```

Comment:
L427-L437 : is this really necessary, given the same thing is implemented in eclipse?